lms/djangoapps/course_structure_api/v0/tests.py

```diff
@@ -336,6 +335,7 @@ def add_block(xblock):
             blocks[unicode(xblock.location)] = {
                 u'id': unicode(xblock.location),
                 u'type': xblock.category,
+                u'parent': None,
```

Comment:
Yes, I see that the parent will be set to `None` in all the tests. However, is there a circumstance in which it will not be set to `None`? Should you test for that scenario?